resources/views/product/partials/queries/options.graphql

```diff
@@ -0,0 +1,55 @@
+{
+    products(filter: { sku: { eq: "{{ $sku }}" } }) {
```

Comment:
It's better to work with a named graphql query and variables here.
It _could_ allow the query to be [cached](https://graphql.org/learn/caching/) by Magento or Client.
Very much **could** i dont think magento does it and neither our client

Reply:
The query is removed as we're getting it from the database now 🚀